package.json

```diff
@@ -9,7 +9,7 @@
     "@material-ui/core": "^4.10.0",
     "@material-ui/icons": "^4.9.1",
     "@svgr/webpack": "4.1.0",
-    "axios": "^0.19.1",
+    "axios": "^0.21.1",
```

Comment:
We can update version later at once. It would be easy to track if issue is related to dependencies.